sdk/cognitiveservices/azure-ai-speech-transcription/tspconfig.yaml

```diff
@@ -0,0 +1,7 @@
+emit:
```

Comment:
This file should not be in this repo. This should be in the rest-api-specs repo.